### PR TITLE
Purchase Deed Packets Using Hometown Data

### PR DESCRIPTION
**Pull request by jandersson**
This loads the town data and looks for `deed-room` and `deed-number` in the blacksmithing section, then runs off to purchase like usual. If they dont exist for the town then the script exits.

### CODE REVIEW

**Comment by rpherbig** [comment]
Why did you add this to the blacksmithing section and not a (new?) mining section?

**Comment by jandersson** [comment]
No special reason, just that they're in the forging society. Want me to making a mining section?

**Comment by jandersson** [comment]
I'm going to add this to a `deeds:` section. I think this is appropriate because deeds are decoupled from crafting disciplines and even materials. You can deed items, wood, and ore, so i think its fitting as its own section. Let me know what you think

**Comment by rpherbig** [comment]
That makes sense to me

File: data/base-crafting.yaml [hunk 1]
```diff
@@ -126,6 +126,8 @@ blacksmithing:
     finisher-full: flask of oil
     finisher-room: 4436
     finisher-number: 6
+    deeds-room: 4433
```
Comment (rpherbig) [inline data/base-crafting.yaml]: Should this be removed now?

Comment (jandersson) [inline data/base-crafting.yaml]: oops, yes it should

File: mining-buddy.lic [hunk 2]
```diff
@@ -32,7 +32,17 @@ class MiningBuddy
   end
 
   def buy_deed_packet
-    order_item(8775, 14)
+    hometown = get_settings.hometown
```
Comment (rpherbig) [inline mining-buddy.lic]: What do you think about moving these few lines (just the settings part) to `setup`?

Comment (jandersson) [inline mining-buddy.lic]: thats fine with me, pass the room number and order number as arguments to `buy_deed_packet`?

Comment (rpherbig) [inline mining-buddy.lic]: Eh, I'd just make them instance vars like the others.

Comment (jandersson) [inline mining-buddy.lic]: Yep, on second thought that looks easier

**Pull request merged**